expression/builtin_string.go

```diff
@@ -2269,6 +2269,10 @@ func (b *builtinRpadUTF8Sig) evalString(row chunk.Row) (string, bool, error) {
 	}
 	padLength := len([]rune(padStr))
 
+	if len(padStr)*targetLength > (1 << 32) {
```

Comment:
I don't know what we are trying to do here, but if we are trying to check if this size fits in an int or 4 bytes, then the comparison should be 

```suggestion
	if len(padStr)*targetLength > (1 << 31) {
```